tests/queries/0_stateless/01889_sql_json_functions.reference

```diff
@@ -77,6 +77,8 @@ SELECT JSON_QUERY('{"array":[[0, 1, 2, 3, 4, 5], [0, -1, -2, -3, -4, -5]]}', '$.
 [0, 1, 4, 0, -1, -4]
 SELECT JSON_QUERY('{"1key":1}', '$.1key');
 [1]
+SELECT JSON_QUERY('{"123":1}', '$.123');
+[1]
```

Comment:
add some tests like
```sql
SELECT JSON_QUERY('{"123":1}', '$[123]');
```
Not sure whether is this a valid path.

Reply:
done